.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.linting.pylintEnabled": true
+}
```

Comment:
Not sure if it was intentional to include this file, since `.vscode/**` was added to `.gitignore`?

Reply:
Not intentional, I think it was added before I added `.vscode/**` to the `.gitignore`.  Thanks for pointing it out.